returnn/training.py

```diff
@@ -149,6 +152,9 @@ def __init__(
         :param time_rqmt:
         :param mem_rqmt:
         :param cpu_rqmt:
+        :param launch_cmd: the command used to launch training jobs
+            Possible values: "mpirun": use mpirun, c.f. https://www.open-mpi.org/doc/v4.0/man1/mpirun.1.php
+                             "torchrun": use torchrun, c.f. https://pytorch.org/docs/stable/elastic/run.html
```

Comment:
It should mention here that this is only used if `horovod_num_processes` is not None.

Reply:
@albet @JackTemaki  I added a sentence stating that 
`:param launch_cmd: the command used to launch training jobs, only used if horovod_num_processes is not None `